.hlint.yaml

```diff
@@ -51,49 +51,7 @@
     - Network.HTTP.StackClient
 
 # Not considered useful hints
-- ignore: {name: "Redundant do"}
```

Comment:
Does HLint now disable "Redundant do" by default?  If not, you probably want to keep this here.  There is zero harm in redundant `do`s and in `*Spec.hs` files you literally always want them.  Consider:

```haskell

spec :: Spec
spec = do
  describe "..." $ do
    it ".." $ do
```
Here it would be foolish to add/remove `do`s when you add/remove additional spec items just for the sake of appeasing HLint.  Not only is it a waste of your time but it also gives you larger diffs and potentially more conflicts when rebasing etc.

Reply:
Yes, Hlint's "Redundant do" is now off-by-default, since HLint 3.1.2.